fix(curriculum):  Step 103 Pyramid Generator Text (#55162)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660f4e74f7fd3f4a99ac2e50.md

The `numbers` array would be `[2, 3]`.

> Directly below the declaration of `numbers`, declare a `shifted` variable, and assign it the result of calling `.shift()` on your `numbers` array. Then, on the next line, print the variable.

Reviewer: We can make the text simpler a bit:

```suggestion
Directly below your `numbers` array, declare a `shifted` variable and assign it the result of calling `.shift()` on the `numbers` array. On the next line, log the `shifted` variable to the console.
```
